### PR TITLE
Make podspec dependencies more granular

### PR DESCRIPTION
**Pull request by itsthejb**
Hi there,

Great lib! We're just starting using this, and noticed that in spite of the current use of subspecs, it would be nice to make them just a shade more granular. In particular, we'd rather not CloudKit unnecessarily, which is currently required in the Standard subspec.

In any case, please let me know what you think.

Thanks!

### CODE REVIEW

**Comment by danthorpe** [comment]
Hi! @itsthejb thanks a lot for the input! I totally agree with you on this, and that the subspecs can be a little bit more granular.

There is an existing issue #112 which is sort of related. Part of the issue is that we have stuff which is not API Extension compatible. And the goal for v2.7 is to combine all the code into one workspace/project where the core `Operations` module is Extension compatible, and have everything else broken up into modules. I think the CocoaPod specs should then mirror this.

Happy to merge this into `development` now, if you are already on the "bleeding edge". However, I would like to sort out all the project configurations as part of 2.7 to avoid so many continual breaking changes. If you can manage for now with v2.6 - then it might just be easier to wait until the changes for #112 are proposed and see what you think?

File: Operations.podspec [hunk 1]
```diff
@@ -121,7 +129,47 @@ session Advanced NSOperations: https://developer.apple.com/videos/wwdc/2015/?id=
...
+  # Subspec which includes CloudKit functionality
+  s.subspec '+CloudKit' do |ss|
+    ss.platforms = { :ios => "8.0", :osx => "10.10" }
+    ss.dependency 'Operations/Standard'
+    ss.frameworks = 'CloudKit'
+    ss.source_files = [
+      'Sources/Features/Shared/CloudCapability.swift',
+      'Sources/Features/Shared/CloudKitOperation.swift',			
+    ]
+  end
```
Comment (danthorpe) [inline Operations.podspec]: This is a duplication in error I think which should be removed. 

Also CloudKit is available on tvOS platform, which will need to be added here, and updated in the excludes from Standard spec.

**Comment by itsthejb** [comment]
Hey there,

Well we've forked for the time being, so no particular urgency at our end. Whatever you think best in the long run.

**Comment by itsthejb** [comment]
Hi there,

Had a chance to make those changes you recommended. Please let me know what you think. I'm afraid a bit too snowed under to look into the changes for the upcoming release, though, so sorry if there's any redundancy.

**Comment by danthorpe** [comment]
@itsthejb thanks for this - sorry it took a little while to merge. Preparing v2.7 now which will include these changes. Thanks!

**Pull request merged**